examples/dependency/demo.yaml

```diff
@@ -8,6 +8,8 @@ spec:
     kind: Foo
     metadata:
       name: source
+    status:
+      key: test
```

Comment:
This should not exist at initial time.

Reply:
Let me comment it out. Having this without typing every time is very convenient.